.github/workflows/ci.yml

```diff
@@ -0,0 +1,68 @@
+name: Node.js CI with PostgreSQL
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build_and_test:
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:latest
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: tbd
+          POSTGRES_DB: mockpfi
+        ports:
+          - 5432:5432
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+
+    strategy:
+      matrix:
+        node-version: ['20.4.0']
+
+    env:
+      ENV: production
+      LOG_LEVEL: info
+      HOST: http://localhost:9000
+      PORT: 9000
+      #SEC_DB_HOST: postgres # Use the service name for the host
+      #SEC_DB_PORT: 5432
+      #SEC_DB_USER: postgres
+      #SEC_DB_PASSWORD: tbd
+      #SEC_DB_NAME: mockpfi
```

Comment:
i believe all of these can be removed as they're already the defaults defined [here](https://github.com/TBD54566975/tbdex-pfi-exemplar/blob/main/src/config.ts#L28-L37) and [here](https://github.com/TBD54566975/tbdex-pfi-exemplar/blob/main/db/scripts/common#L8-L13)

Reply:
done